File: Eigenvector/N5_power_method/import_dat.cpp
#include <iostream>
#include <fstream>
#include <sstream>
#include <vector>
#include <string>

int main() {
    std::ifstream file("N4_permute.dat");
    std::string line;

    while (std::getline(file, line)) {
        std::istringstream iss(line);
        std::vector<int> elements;

        int num;
        while (iss >> num) {
            elements.push_back(num);
        }

        // Now 'elements' contains the elements of the current line
        // You can process these elements as needed
        for (int element : elements) {
            std::cout << element << " ";
        }
        std::cout << std::endl;
    }

    return 0;
}

File: Eigenvector/N5_power_method/import_dat.py
with open("N4_permute.dat", "r") as file:
    for line in file:
        elements = list(map(int, line.split()))
        
        # Now 'elements' contains the elements of the current line
        # You can process these elements as needed
        for element in elements:
            print(element, end=" ")
        print()

